lib/_kernel.scm

```diff
@@ -4279,6 +4279,12 @@ end-of-code
             scheme-object   ;; device
    "___os_device_stream_open_process"))
 
+(define-prim ##os-device-open-raw
```

Comment:
Is the concern the name `fd`? You are right, will call the argument something different to avoid confusion.